.idea/.name

```diff
@@ -0,0 +1 @@
+student20_pop_fe2
```

Comment:
Should we be including anything from ".idea" directory? Maybe others want to setup different settings in their IDE (code Styles) or use a different IDE altogether.